src/content/docs/develop/index.mdx

```diff
@@ -46,14 +65,138 @@ for a more detailed explanation.
+  cargo="cargo tauri [android|ios] dev"
+/>
 
-Once Rust has finished building, the webview opens, displaying your web app. You can make changes to your web app, and if your tooling enables it, the webview should update automatically, just like a browser. When you make changes to your Rust files, they are rebuilt automatically, and your app automatically restarts.
+The first time you run this command, the Rust package manager may need **several minutes** to download and build all the required packages.
+Since they are cached, subsequent builds are much faster, as only your code needs rebuilding.
 
-Note that Tauri's APIs only work in your app windows, so once you start using them you won't be able to open your frontend in your system's browser anymore.
+#### Development Server
 
-:::note[About Cargo.toml and Source Control]
+To run your application on mobile you must expose your development server in your public network.
+Usually this can be done by providing a `--host` CLI option on your [beforeDevCommand](../reference/config.md#beforedevcommand)
+or defining a host in the UI framework or JavaScript bundler configuration.
 
-In your project repository, you **SHOULD** commit the `src-tauri/Cargo.lock` along with the `src-tauri/Cargo.toml` to git because Cargo uses the lockfile to provide deterministic builds. As a result, it is recommended that all applications check in their `Cargo.lock`. You **SHOULD NOT** commit the `src-tauri/target` folder or any of its contents.
+Check your framework's setup guide for more information.
 
+:::note
+Projects created with [create-tauri-app](https://github.com/tauri-apps/create-tauri-app) configures
+your development server for mobile dev out of the box.
 :::
+
+#### Device Selection
+
+By default the mobile dev command tries to run your application in a connected device,
+and fallbacks to prompting you to select a simulator to use.
+To define the run target upfront you can provide the device or simulator name as argument:
+
+<CommandTabs
+  npm="npm run tauri ios dev 'iPhone 15'"
+  yarn="yarn tauri ios dev 'iPhone 15'"
+  pnpm="pnpm tauri ios dev 'iPhone 15'"
+  cargo="cargo tauri ios dev 'iPhone 15'"
+/>
+
+#### Using Xcode or Android Studio
+
+Alternatively you can choose to use Xcode or Android Studio to develop your application.
+This can help you troubleshoot some development issues by using the IDE instead of the command line tools.
+To open the mobile IDE instead of running on a connected device or simulator, use the `--open` flag:
+
+<CommandTabs
+  npm="npm run tauri [android|ios] dev --open"
+  yarn="yarn tauri [android|ios] dev --open"
+  pnpm="pnpm tauri [android|ios] dev --open"
+  cargo="cargo tauri [android|ios] dev --open"
+/>
+
+:::caution
+To use Xcode or Android Studio the Tauri CLI process **must** be running and **cannot** be killed.
+It is recommended to use the `tauri [android|ios] dev --open` command and keep the process alive until you close the IDE.
+:::
```

Comment:
I think it's also worth noting, in the case of XCode at least, that the `Build Rust Code` phase is ran within a sanitized environment that might not include the tauri binary location, resulting in a fail. (May happen depending on how the toolchain was installed, see [#6573](https://github.com/tauri-apps/tauri/issues/6573))

Easiest fix I found for me using cargo was to symlink the cargo binary to `/usr/local/bin/cargo`

Reply:
will do, thanks!